openapi/Swarm.yaml

```diff
@@ -853,20 +853,38 @@ paths:
 
   "/health":
     get:
-      summary: Get health of node
-      description: This endpoint is available on the main API only if the node is spawned with the `--restricted` flag.
+      summary: Get node overall health Status
+      description: |
+        Health Status will indicate node healthiness (healthiness is k8s equivalent for liveness).
```

Comment:
I find the "(healthiness is k8s equivalent for liveness)" part to be irrelevant since it assumes that the consumer of the API is familiar with the k8s, which might not be the case.

Reply:
I assume that main purpose behind these two endpoints is for orchestration solutions (like k8s). That's why I wanted to be more descriptive with this description.

However, that part from the docs has been removed in recent change.